pkg/trace/api/endpoints.go

```diff
@@ -80,4 +83,20 @@ var endpoints = []endpoint{
 		Pattern: "/v0.6/stats",
 		Handler: func(r *HTTPReceiver) http.Handler { return http.HandlerFunc(r.handleStats) },
 	},
+	{
+		Pattern: "/appsec",
+		Handler: func(r *HTTPReceiver) http.Handler {
+			return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+				req.URL.Path = strings.TrimPrefix(req.URL.Path, "/appsec")
```

Comment:
Why did you do this? Did you mean to be using https://golang.org/pkg/net/http/#StripPrefix instead?

Reply:
I tried `http.StripPrefix` indeed, but it doesn't work from here: we end up either with 301 Moved Permanently with a request to `/appsec`; or 404 Not Found with a request to `/appsec/`.

Reply:
Could it be that you might have been doing something wrong? Check https://play.golang.org/p/PqdIdFPWchG

Reply:
Now fixed, but only with `/appsec/` instead of `/appsec` so that it gets stripped to `/` and the following AppSec ServeMux can properly match it.

For the record: the problem I had was coming from a mix if StripPrefix when it returns the empty string, and the AppSec ServeMux which wasn't happy about it because it expects "/" and not "".